File: eRecipes/eRecipes.Model/Transakcija14072025.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eRecipes.Model
{
    public class Transakcija14072025
    {
        public int Transakcija14072025Id {  get; set; }
        public int KorisnikId { get; set; }
        public Korisnik Korisnik { get; set; }
        public float Iznos {  get; set; }
        public DateTime DatumTransakcije { get; set; }
        public string Opis {  get; set; }
        public int KategorijaTransakcije14072025Id { get; set; }
        public KategorijaTransakcije14072025 KategorijaTransakcije14072025 { get; set; }
        public string Status { get; set; }
    }
}

File: eRecipes/UI/erecipes_mobile/lib/models/transakcija14072025.dart

import 'package:erecipes_mobile/models/kategorija_transakcije14072025.dart';
import 'package:erecipes_mobile/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'transakcija14072025.g.dart';

@JsonSerializable()
class Transakcija14072025 {
  int? transakcija14072025Id;
 int? korisnikId;
 Korisnik? korisnik;
 int? kategorijaTransakcije14072025Id;
 KategorijaTransakcije14072025? kategorijaTransakcije14072025;
 double? iznos;
 DateTime? datumTransakcije;
 String? opis;
 String? status;

  Transakcija14072025(
    this.korisnikId,
    this.iznos,
    this.datumTransakcije,
    this.opis,
    this.kategorijaTransakcije14072025Id,
    this.status
  );

  factory Transakcija14072025.FromJson(Map<String, dynamic> json) =>
     _$Transakcija14072025FromJson(json);

  Map<String, dynamic> toJson() => _$Transakcija14072025ToJson(this);
}
